src/main/java/com/conveyal/datatools/manager/controllers/api/FeedSourceController.java

```diff
@@ -141,6 +149,11 @@ private static void validate(Request req, FeedSource feedSource) {
         if (feedSource.retrieveProject() == null) {
             validationIssues.add("Valid project ID must be provided.");
         }
+        for (String labelId: feedSource.labelIds) {
+            if (Persistence.labels.getByIds(feedSource.labelIds).size() != feedSource.labelIds.size()) {
+                validationIssues.add("All labels assigned to feed must exist.");
+            }
+        }
```

Comment:
This for loop is now unneccessary. It can be rewritten as follows:

```suggestion
        if (Persistence.labels.getByIds(feedSource.labelIds).size() != feedSource.labelIds.size()) {
            validationIssues.add("All labels assigned to feed must exist.");
        }
```

Reply:
Good catch thank you! Slept through that one. Fixed in 4e1ce1f